variables.tf

```diff
@@ -135,8 +135,50 @@ variable "sns_topic_encryption_enabled" {
   description = "Set this to `false` to disable encryption on the sns topic. Defaults to true"
 }
 
-variable "sns_topic_encryption_key_arn" {
+variable "sns_topic_key_arn" {
   type        = string
   default     = ""
   description = "The ARN of an existing KMS encryption key to be used for the SNS topic"
 }
+
+variable "use_existing_cross_account_iam_role" {
+  type        = bool
+  default     = false
+  description = "Set this to true to use an existing IAM role for cross-account access"
+}
+
+variable "iam_role_arn" {
```

Comment:
Should we also require the user to supply the iam role name?

Reply:
I am not sure how much value there is to having the role name for an input value. I've found that most of the time the ARN is easier to supply. Plus most of our TF modules are for ARN, so I was trying to follow suit. Lastly, its the role ARN we need to supply for our lacework integration resource, as well as the cloudwatch filter and kinesis firehose.